src/plugins/tagfilter/tagfilter-en.hbs

```diff
@@ -981,6 +982,246 @@
 &lt;/section></code></pre>
 </details>
 
+
+<section id="filteredFinder" class="wb-tagfilter provisional wb-filter" data-wb-tagfilter='{ "live": false }' data-wb-filter='{
+		"selector": "section",
+		"section": ".wb-tagfilter-items",
+		"uiTemplate": "#searchFilter2",
+		"live": false
+	}'>
+	<h2>Example 4 - Finder with "or" behavior updating with a button</h2>
+
+	<div class="row">
+	<search id="searchFilter2" class="col-xs-12 col-md-3 pull-right">
+		<h3>Filters</h3>
+		<div>
+			<p class="wb-fltr-info mrgn-tp-0"><span data-nbitem></span> results out of <span data-total></span></p>
+		</div>
+
+		<form method="get">
+			<div class="form-group">
+				<label class="text-muted" for="customSearchUI2">Search for a term</label>
+				<input type="search" class="form-control full-width" id="customSearchUI2">
+			</div>
+
+			<div class="form-group">
+			<fieldset>
+				<legend>Audience</legend>
+
+				<ul class="list-unstyled">
+					<li class="checkbox"><label><input type="checkbox" name="audience" value="student" class="wb-tagfilter-ctrl" data-wb-tagfilter='{"group": "orConditionGroup"}' /> Student</label></li>
+					<li class="checkbox"><label><input type="checkbox" name="audience" value="worker" class="wb-tagfilter-ctrl" data-wb-tagfilter='{"group": "orConditionGroup"}' /> Worker</label></li>
+					<li class="checkbox"><label><input type="checkbox" name="audience" value="senior" class="wb-tagfilter-ctrl" data-wb-tagfilter='{"group": "orConditionGroup"}' /> Senior</label></li>
```

Comment:
The ``bootlint`` task is failing on this checkbox structure (see [error E018](https://github.com/twbs/bootlint/wiki/E018)).

**Possible workarounds:**
* Use  a group of ``div``s instead of a list
* Remove ``class="checkbox"`` from the ``li`` elements and wrap ``<div class="checkbox">...</div>`` around the labels.
* Modify the [``bootlint:all`` task](https://github.com/wet-boew/wet-boew/blob/01060c7bb1e641afbdc7f1f313c276eef294eb68/Gruntfile.coffee#L974-L994) to ignore the error on the tag filter plugin's demo pages
  ```coffee
  "E018" # Incorrect markup used with the `.radio` class. The correct markup structure is `.radio>label>input[type="radio"]`
  ```
  **Note:** Might want to create a separate ``tagfilter`` bootlint task to avoid exempting it globally... btw I was unable to get [``relaxerror``'s file targeting feature](https://www.npmjs.com/package/grunt-bootlint#optionsrelaxerror) working correctly in the ``all`` task :S

**PS:**
[Bootlint is archived](https://github.com/twbs/bootlint)... but still technically makes sense to have in WET/GCWeb given that both are based on ancient versions of Bootstrap :P.

Reply:
Whoops - nevermind. I misinterpreted what was up. Please ignore this.